src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -101,7 +101,7 @@ public PublicClientApplicationBuilder WithIosKeychainSecurityGroup(string keycha
         /// </summary>
         /// <param name="enableBroker"></param>
         /// <returns></returns>
-        private PublicClientApplicationBuilder WithBroker(bool enableBroker)
+        public PublicClientApplicationBuilder WithBroker(bool enableBroker)
```

Comment:
>public PublicClientApplicationBuilder WithBroker(bool enableBroker) [](start = 8, length = 67)

I understand why we have this, will it be possible to set this by default to true? allowing people to skip passing in the bool value?

Reply:
sure. let's talk friday. esp if we are only routing to broker when msauth:// is appended  to the auth code, there shouldn't be any change in behavior for the customer. need clarify on when msauth:// will be present first. :) 

---
In reply to: [300211688](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1251#discussion_r300211688) [](ancestors = 300211688)